src/main/kotlin/app/revanced/cli/aligning/ZipAligning.kt

```diff
@@ -0,0 +1,19 @@
+package app.revanced.cli.aligning
+
+import app.revanced.cli.command.MainCommand
+import app.revanced.cli.command.MainCommand.logger
+import app.revanced.utils.signing.align.ZipAligner
+import java.io.File
+
+object ZipAligning {
```

Comment:
Call this "Aligning" the same way "Signing" is called, then we can merge 👍

Reply:
Done.